test/e2e/tests/advanced-settings.spec.js

```diff
@@ -0,0 +1,76 @@
+const { strict: assert } = require('assert');
+const { convertToHexValue, withFixtures } = require('../helpers');
+
+describe('Advanced Settings', function () {
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+
+  it('should show conversion on test network when activated', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        await driver.clickElement('.account-menu__icon');
+        await driver.clickElement({ text: 'Settings', tag: 'div' });
+        await driver.clickElement({ text: 'Advanced', tag: 'div' });
+
+        await driver.clickElement(
+          '[data-testid="advanced-setting-show-testnet-conversion"] label',
+        );
+
+        await driver.clickElement(
+          '.settings-page__header__title-container__close-button',
+        );
+
+        const secondaryCurrency = await driver.isElementPresent(
+          '[data-testid="eth-overview__secondary-currency"]',
+        );
+        assert.equal(
+          await secondaryCurrency,
```

Comment:
Do you have to await here too, even though you did so on line 40? Also instead of checking for `isElementPresent` should we check for the text we expect to be present? In this case `USD` right?

Reply:
great catch! I removed the `await` for both `it` blocks as there was not needed. Thank you @adonesky1!

For the `isElementPresent`, I believe there is no difference. I chose this approach as we already had a function on the driver, and was straightforward to apply, as this element represents the secondary currency. Would you recommend me to write another function for `isTextPresent` in the driver file and use this one instead? Ty!!

Reply:
@seaona I guess I'm just thinking of something like:
```
const secondaryCurrency = await driver.findElement(
          '[data-testid="eth-overview__secondary-currency"]',
        );
 assert.equal(secondaryCurrency.getText(), 'USD')
```

or something like that?

Reply:
oh the thing is that `data-testid="eth-overview__secondary-currency` element only appears when the functionality is active, so in the case is deactivated, the `findElement` would fail as this element is not there anymore, so this `findElement` would fail @adonesky1 

![image](https://user-images.githubusercontent.com/54408225/183944499-7ab24d02-d786-4cc7-bd51-de8159b9016b.png)

Reply:
Could we use the `.findElement` pattern in the `should show conversion on test network when activated` test and use `isElementPresent` on `should not show conversion on test network when deactivated`? Or is that a testing anti-pattern, since you should use the same condition for on and off?

Reply:
I think it is okay to do this! I will update it right now @adonesky1 thank you!!!